magento/pwa-studio
# Mini Cart V2 / Shopping Bag header component.

## Description

Added header section to the new Mini Cart / Shopping Bag component.

**Should be merged only after #2506** 

## Related Issue
Closes [PWA-116](https://jira.corp.magento.com/browse/PWA-116)

### Verification Stakeholders
@jimbo 
@soumya-ashok 
@schensley 

### Verification Steps

1. run yarn watch:venia.
2. Add an item to the cart and click the new cart button.
3. We should be able to see the header component.

## Screenshots / Screen Captures (if appropriate)
![image](https://user-images.githubusercontent.com/35203638/85595795-c9396280-b60e-11ea-998b-39db7bf8f3f7.png)

## Checklist

* **Should be merged only after #2506**
